packages/jest-diff/package.json

```diff
@@ -10,7 +10,12 @@
   "dependencies": {
     "chalk": "^2.0.1",
     "diff": "^3.2.0",
+    "diff-match-patch": "1.0.0",
     "jest-get-type": "^21.2.0",
     "pretty-format": "^21.2.1"
+  },
+  "devDependencies": {
```

Comment:
dev deps are at the bottom level package.json

Reply:
Thank you very much!